CommonInstall/util_init.pl

```diff
@@ -213,7 +213,7 @@ ($)
 			chop($rval);
 			$rval="ES".$rval;
 			if ( -e "/etc/redhat-release" ) {
-				if (!system("grep -qi centos /etc/redhat-release") || !system("grep -qi rocky /etc/redhat-release") || !system("grep -qi almalinux /etc/redhat-release") || !system("grep -qi circle /etc/redhat-release")) {
+				if (!system("grep -qi centos /etc/redhat-release") || !system("grep -qi rocky /etc/redhat-release") || !system("grep -qi almalinux /etc/redhat-release") || !system("grep -qi circle /etc/redhat-release" || !system("grep -qi oracle /etc/oracle-release")) {
```

Comment:
While testing we hit a regression here at this point when running on a RHEL systems.
`grep: /etc/oracle-release: No such file or directory`

I think we need add a check for the file before greping it. Also there is a missing `)`

```diff
...
- !system("grep -qi circle /etc/redhat-release" || !system("grep -qi oracle /etc/oracle-release"))
+ !system("grep -qi circle /etc/redhat-release") || ( -e "/etc/oracle-release" && !system("grep -qi oracle /etc/oracle-release"))
```

Reply:
I’ve added the missing `)` and the file existence check.

https://github.com/Mr-Tao/eth-fast-fabric/blob/106eff82d762c5d6f665d8cd91dabcb12d7716c5/CommonInstall/util_init.pl#L216